packages/imagemagick7.rb

```diff
@@ -64,5 +64,12 @@ def self.build
 
   def self.install
     system 'make', "DESTDIR=#{CREW_DEST_DIR}", 'install'
+    system "ln -s #{CREW_LIB_PREFIX}/libMagickWand-7.Q16HDRI.so.6 #{CREW_LIB_PREFIX}/libMagickWand-7.Q16.so.6"
+    system "ln -s #{CREW_LIB_PREFIX}/libMagickCore-7.Q16HDRI.so.6 #{CREW_LIB_PREFIX}/libMagickCore-7.Q16.so.6"
+  end
+
+  def self.remove
+    system "rm #{CREW_LIB_PREFIX}/libMagickWand-7.Q16.so.6"
+    system "rm #{CREW_LIB_PREFIX}/libMagickCore-7.Q16.so.6"
```

Comment:
We will also need new binaries so please remove the existing `binary_url` and `binary_sha256` sections.

Reply:
I will apply it to `imagemagick6.rb` too